### DISCUSSION TITLE
Release 1.0.0

### DISCUSSION THREAD

**Discussion by nigelgbanks** [Islandora-Devops/isle-buildkit#193] [Release]
Determining what is to be included before we make 1.0.0 release.

**Comment by nigelgbanks** [comment]
From the zoom discussion 

PRs need to be identified and merged prior to April
Identify and create a list
- https://github.com/Islandora-Devops/isle-buildkit/pull/187
- https://github.com/Islandora-Devops/isle-buildkit/pull/188 
- https://github.com/Islandora-Devops/isle-buildkit/pull/185
- https://github.com/Islandora-Devops/isle-buildkit/pull/183

**Reply by g7morris** [reply]
Hi @nigelgbanks As you can see I'm following up on a bunch o' things from the meeting and your list above. 

Hoping we can check in during the week asynchronously here as decided in the last meeting for who needs to do what, when, how etc. 

---

### Actions
I've gone ahead and created the [March / April 2022 - isle-buildkit 1.0.0 release](https://github.com/Islandora-Devops/isle-buildkit/projects/2) project with various columns. Please go ahead and move things around as you see fit / if / when you have time. Thanks!

---

### Questions

I do have some open questions re the following

* https://github.com/Islandora-Devops/isle-buildkit/issues/172
* https://github.com/Islandora-Devops/isle-buildkit/issues/174

From what I can see of the list above only https://github.com/Islandora-Devops/isle-buildkit/pull/183 remains?

---

### To dos for release

* https://github.com/Islandora-Devops/isle-buildkit/issues/168
* https://github.com/Islandora-Devops/isle-buildkit/issues/169
* https://github.com/Islandora-Devops/isle-buildkit/pull/183
* https://github.com/Islandora-Devops/isle-buildkit/issues/184
* https://github.com/Islandora-Devops/isle-buildkit/issues/186 (_this looks done?_)

---

Please also feel free to reassign any ticket you see fit to me for testing with steps for either replication or the desired outcome.

Cheers, 
Gavin

**Reply by nigelgbanks** [reply]
Good stuff, I think I've replied to all the above.  I've I get some spare cycles I'll have a look at doing #174 as well.

**Comment by g7morris** [comment]
Sorry @nigelgbanks one more thing that just popped up. 

I'm seeing two Prs / issues that seemingly impact `isle-dc`, I'm assuming we aren't putting these on the list for the release due to time and complexity? 

* `install profile` #170 
* `Adding toggle for installing islandora_defaults...` #145 
  * left a question here and moved into the questions column for review

Labeling them as `docker-compose`, `drupal` and/or `isle-dc` for now so folks have a better gauge.

**Reply by nigelgbanks** [reply]
I've closed #145  as it was dead in the water, #170 probably won't make it into the release, it's hard to say. It'll depend on the progress I make on automation around the sandbox deployments. If it goes fast I'll raise a pull request here or update that one with the changes required to run sandbox as the demo image.

**Comment by g7morris** [comment]
Wahoo! :tada:  @nigelgbanks cut the new official [1.0.0](https://github.com/Islandora-Devops/isle-buildkit/releases/tag/1.0.0) isle-buildkit Release. Well done Nigel you are a :star: 

See you in June for the next round. :grin: 

Closing this discussion
